engine/access/rpc/backend/script_executor.go

```diff
@@ -167,13 +180,26 @@ func (s *ScriptExecutor) checkDataAvailable(height uint64) error {
 	if err != nil {
 		return fmt.Errorf("could not get lowest indexed height: %w", err)
 	}
+
 	if height < lowestHeight {
-		return fmt.Errorf("%w: block is before lowest indexed height", storage.ErrHeightNotIndexed)
+		return ErrIncompatibleNodeVersion
 	}
 
 	if height > s.maxCompatibleHeight.Load() || height < s.minCompatibleHeight.Load() {
 		return fmt.Errorf("%w: node software is not compatible with version required to executed block", storage.ErrHeightNotIndexed)
```

Comment:
This one, should be updated to `ErrIncompatibleNodeVersion` since it's able version compatibility.